saviSimulation/savi/jason_processing/UxVBehavior.java

```diff
@@ -19,13 +19,13 @@
 	//String ID; -- Note: moved to superclass 
 	//String type; -- same
 	//SyncAgentState agentState; --same
-	PVector initialPosition;
-	PVector position;
-	double speedVal;
-	double compasAngle;
-	ArrayList<CameraPerception> visibleItems;
-	double time;	
-	double wifiProbWorking;
+	protected PVector initialPosition;
+	protected PVector position;
+	protected double speedVal;
+	protected double compasAngle;
+	protected ArrayList<CameraPerception> visibleItems;
+	protected double time;	
+	protected double wifiProbWorking;
 	
```

Comment:
Not a fan of having position duplicated between UxV (from WorldObject) and UxVBehaviour. I think all these properties are "physical" and should be associated with the WorldObject / UxV models which to me represent the physical reality of these things as opposed to their behaviour.

Reply:
Agreed

Reply:
Fixed